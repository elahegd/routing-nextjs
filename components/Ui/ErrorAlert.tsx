import React, { FC } from "react";

import classes from "./ErrorAlert.module.css";

interface ErrorAlertProps {
  children: React.ReactNode;
}
const ErrorAlert: FC<ErrorAlertProps> = ({ children }) => {
  return <div className={classes.alert}>{children}</div>;
};

export default ErrorAlert;
