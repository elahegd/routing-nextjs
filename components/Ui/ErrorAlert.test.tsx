import React from "react";
import { render } from "@testing-library/react";

import ErrorAlert from "./ErrorAlert";

describe("Render and check props", () => {
  it("Should render with children props", () => {
    const { asFragment } = render(<ErrorAlert>hey Alert</ErrorAlert>);

    expect(asFragment()).toMatchSnapshot();
  });
});
