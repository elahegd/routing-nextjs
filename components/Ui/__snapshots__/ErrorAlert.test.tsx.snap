// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render and check props Should render with children props 1`] = `
<DocumentFragment>
  <div
    class="alert"
  >
    hey Alert
  </div>
</DocumentFragment>
`;
